R/matrixfit.R

```diff
@@ -3,8 +3,22 @@ bootstrap.meanerror <- function(data, R=400, l=20) {
   return(apply(bootit$t, 2, sd))
 }
 
+#' Correlator matrix model.
```

Comment:
how is this going to interfere with the documentation in hadron/man ? Why do we need to use roxygen here?

Reply:
Roxygen will generate the `man/*.Rd` files for each documented function in the `R/*.R` files. It won't overwrite files it did not create. Therefore one could just run `devtools::document()` and then add the generated files to the Git repository. Instead of modifying the `Rd` file, one would update the documentation in the source code.

We do not have to use it. I use this “literate programming” style for all other languages, so Roxygen feels natural to me. However, since `hadron` does not use it yet, I can also just write `Rd` files and check those in.